core/src/trezor/ui/layouts/tr/__init__.py

```diff
@@ -13,12 +13,80 @@
     ExceptionType = BaseException | Type[BaseException]
 
 
+class DoublePressHandler:
```

Comment:
I'd prefer to handle the double press directly in the `Button` component unless there's reason not to.

Reply:
Done in [9d58d42](https://github.com/trezor/trezor-firmware/pull/2233/commits/9d58d42848c1f531f88c2751b3cf4a42f376ad4c)

One "issue" is still that the first click is registered as a "normal click" and the appropriate button changes its visual state to `Pressed`. I am changing the visual state back to `Released` as soon as we register a double-click, but the quick visual state change can be disturbing for the user.

Can it be somehow resolved without bringing other disadvantages and complexity? Or is the current state OK?